geps/gep-1897.md

```diff
@@ -0,0 +1,171 @@
+# GEP-1897: TLS from Gateway to Backend for ingress
+
+* Issue: [#1897](https://github.com/kubernetes-sigs/gateway-api/issues/1897)
+* Status: Provisional
+
+## TLDR
+
+This document specifically addresses the topic of conveying HTTPS from the Gateway
+dataplane to the backend (backend TLS termination), and intends to satisfy the single
+use case “As a client implementation of Gateway API, I need to know how to connect to
+a backend pod that has its own certificate”. TLS configuration can be a nebulous topic,
+so in order to drive resolution this GEP focuses only on this single piece of functionality.
+
+## Immediate Goals
+
+1. The solution must satisfy the following use case: the backend pod has its own
+certificate and the gateway implementation client needs to know how to connect to the
+backend pod. (Use case #4 in [Gateway API TLS Use Cases](#references))
+2. In terms of the Gateway API personas, only the application developer persona in this
+solution. The application developer should control the gateway to backend TLS settings,
+not the cluster operator, as requiring a cluster operator to manage certificate renewals
+and revocations would be extremely cumbersome.
+3. The solution should consider client certificate settings used in the TLS handshake **from
+Gateway to backend**, such as TLS versions and cipher suites. (Use case #5 in [Gateway API TLS Use Cases](#references))
+
+## Longer Term Goals
+
+These are worthy goals, but may need a different GEP for proper attention.
+
+1. TCPRoute use cases (completed by GA)
+2. Mutual TLS use cases
+3. Service mesh use cases
+
+## Non-Goals
+
+These are worthy goals, but will not be covered by this GEP.
+
+1. Changes to the existing mechanisms for edge or passthrough TLS termination
+2. Providing a mechanism to decorate multiple route instances
+3. TLSRoute use cases
+4. UDPRoute use cases 
+5. Termination of TLS for HTTP routing (#1 in [Gateway API TLS Use Cases](#references))
+6. HTTPS passthrough use cases (#2 in [Gateway API TLS Use Cases](#references))
+7. Termination of TLS for non-HTTP TCP streams (#3 in [Gateway API TLS Use Cases](#references))
+8. Controlling certificates used by more than one workload (#6 in [Gateway API TLS Use Cases](#references))
+9. Client certificate settings used in TLS **from external clients to the
+Listener** (#7 in [Gateway API TLS Use Cases](#references))
+10. Providing a mechanism for the cluster operator to override gateway to backend TLS settings.
+
+## Overview - what do we want to do?
+
+Given that the current ingress solution specifies **edge** TLS termination (from the client to
+the gateway), and how to handle **passthrough** TLS (from the client to the backend pod), this
```

Comment:
Note that 'passthrough' is covered by TLSRoute ( routing via SNI without edge termination ). 

Gateway communicating with a backend using TLS has nothing to do with edge TLS termination - it is just a property of the backend that it requires TLS. Even if the original internet request was plain HTTP - the gateway would still have to use TLS to the backend ( because typically that's all that the backend accepts ).

That's why 're-encryption' or 'passthrough' are not right terms here.

Reply:
This section is saying that TLS termination at the _Listener_ is already done, as is TLS Passthrough (with TLSRoute). It's entirely appropriate to use those terms here in those two cases.